build_mt.py

```diff
@@ -177,38 +177,29 @@ def correct_inds(halo_ids, N_halos_slabs, slabs, inds_fn):
 
     return ids
 
-def get_mt_info(fns, fields, minified, inds_fn):
+def get_mt_info(fns_load, fields, minified):
     '''
     Load merger tree and progenitors information
     '''
-
-    # selected files to load
-    fns_load = [fns[i] for i in inds_fn]
-
-    # if we are loading all progenitors and not just main
-    if "Progenitors" in fields:
-        merger_tree, Progs = simple_load(fns_load, fields=fields)
-    else:
-        merger_tree = simple_load(fns_load, fields=fields)
-
-    # turn data into astropy table
-    Merger = Table(merger_tree, copy=False)
-    Merger.add_column('ComovingDistance', copy=False)
```

Comment:
I don't think this adds a column named `ComovingDistance`.  Instead, it adds a string-valued column with that string in every row!

Reply:
I usually do `Merger.add_column(np.empty(N, dtype=blah), copy=False, name='ComovingDistance')`.  There may be a more elegant way to add an empty column (i.e. no memset), but I like being explicit.